service/gradle/dependencies.gradle

```diff
@@ -20,9 +20,8 @@ dependencies {
   implementation group: "io.opencensus", name: "opencensus-contrib-http-jaxrs", version: "0.28.3"
 
   // Get stairway via TCL
-  implementation("bio.terra:terra-common-lib:0.0.66-SNAPSHOT") {
-    exclude group: "org.broadinstitute.dsde.workbench", module: "sam-client_2.12"
-  }
+  implementation group: "bio.terra", name: "terra-common-lib", version: "0.0.66-SNAPSHOT"
```

Comment:
I think we probably still need to exclude `sam-client_2.12"` from these 3 dependencies, I saw the old version get pulled into `gradle.lockfile`.

Reply:
OK, I'll bring those back - did not see them used. Thanks!